src/main/java/seedu/duke/template/RcTemplate.java

```diff
@@ -0,0 +1,56 @@
+package seedu.duke.template;
+
+import seedu.duke.component.Capacitor;
+import seedu.duke.component.Resistor;
+
+
+public class RcTemplate extends Template {
+    private static final String RC_TEMPLATE = "\t+---R-----C---+\n"
+            + "\t|             |\n"
+            + "\t|             |\n"
+            + "\t+----+V_ac+---+\n";
+
+    private Resistor resistor;
+    private Capacitor capacitor;
+
+    public RcTemplate(double resistance, double capacitance, double powerSupply) {
+        super(powerSupply);
+        capacitor = new Capacitor(capacitance);
+        resistor = new Resistor(resistance);
+    }
+
+    /**
+     * Returns capacitor object, an attribute of the instance of RcTemplate.
+     *
+     * @return capacitor, an instance of the Capacitor class.
+     */
+
```

Comment:
Consider removing the line between the JavaDoc and method? From the coding standard: https://se-education.org/guides/conventions/java/basic.html#comments
> No blank line between the documentation block and the method/class
```suggestion
```

Reply:
Got it, thanks for pointing out Wira!